daemon/src/model/cfd.rs

```diff
@@ -2517,6 +2766,26 @@ mod tests {
     }
 
     impl Dlc {
+        fn with_lock(
+            mut self,
+            lock_tx: PartiallySignedTransaction,
+            descriptor: Descriptor<PublicKey>,
+            amount_taker: Amount,
+            amount_maker: Amount,
+            taker_address: Address,
+            maker_address: Address,
+            identity_sk: SecretKey,
+        ) -> Self {
+            self.lock = (lock_tx.extract_tx(), descriptor);
+            self.taker_lock_amount = amount_taker;
+            self.maker_lock_amount = amount_maker;
+            self.identity = identity_sk;
+            self.taker_address = taker_address;
```

Comment:
I don't think the addresses should matter here, but since we use them to get the script for the respective party I was not sure and added them from the build params.

Reply:
Answer myself: The addresses are definitely needed because we use the script pubkey of the address when building the close transaction.